<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionSelection"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="separator"/>
    <addaction name="actionPen_Color"/>
    <addaction name="actionPen_Size"/>
   </widget>
   <widget class="QMenu" name="menuOperating_Modes">
    <property name="title">
     <string>Operating Modes</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuReports">
    <property name="title">
     <string>Reports</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuOperating_Modes"/>
   <addaction name="menuDatabase"/>
   <addaction name="menuReports"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionPen_Color"/>
   <addaction name="actionPen_Size"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="separator"/>
   <addaction name="actionSelection"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEdit"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/add-document.svg</normaloff>:/Icon/add-document.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/folder-open.svg</normaloff>:/Icon/folder-open.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/Save.png</normaloff>:/Icon/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/Save As.png</normaloff>:/Icon/Save As.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/circle-xmark.svg</normaloff>:/Icon/circle-xmark.svg</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/cut.png</normaloff>:/Icon/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/Copy.png</normaloff>:/Icon/Copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/paste.png</normaloff>:/Icon/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/zoom-in.svg</normaloff>:/Icon/zoom-in.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/zoom-out.svg</normaloff>:/Icon/zoom-out.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/Undo.png</normaloff>:/Icon/Undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/Redo.png</normaloff>:/Icon/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/edit-button.png</normaloff>:/Icon/edit-button.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionPen_Color">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/palette.svg</normaloff>:/Icon/palette.svg</iconset>
   </property>
   <property name="text">
    <string>Pen Color</string>
   </property>
  </action>
  <action name="actionPen_Size">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/icons8-pen-40.png</normaloff>:/Icon/icons8-pen-40.png</iconset>
   </property>
   <property name="text">
    <string>Pen Size</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Icon/square-dashed.svg</normaloff>:/Icon/square-dashed.svg</iconset>
   </property>
   <property name="text">
    <string>Selection</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Res.qrc"/>
 </resources>
 <connections/>
</ui>
